eng/helix/helix.proj

```diff
@@ -22,7 +22,10 @@
     <IsExternal>true</IsExternal>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
     <HelixAccessToken Condition="'$(_UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
-
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>5.0.100-preview.6.20310.4</DotNetCliVersion>
```

Comment:
Okay looks like hardcoding gets us almost there, @dougbu do you know why $(NETCoreSdkVersion) isn't defined in helix.proj, or I guess more importantly, what's the best way to flow that in, do we have access to that variable or the sdk version in yaml so I guess we can just explicitly pass it into the helix jobs if so?

Reply:
Looking at your Yaml, if you trust that $(NETCoreSdkVersion) you were talking about yesterday is set when that build is invoked (it was hard to follow its flow) you can really really force the issue on the command line that builds your helix .proj with a simple /p:DotNetCliVersion=$(NETCoreSdkVersion)

Reply:
I don't know where `$(NETCoreSdkVersion)` comes from though I suspect it's in the Helix SDK somewhere. Would check there.

Reply:
I believe it actually comes from the .NET Core SDK being used, not Arcade, via Microsoft.NETCoreSdk.BundledVersions.props.

Reply:
Okay so I guess that's the issue, since helix.proj is using the Helix.SDK project, NETCoreSdkVersion isn't defined since its coming from `Sdk="Microsoft.Net.sdk"` Its also not in the yaml, so feeding it in explicitly doesn't work.  Is there an another easy way for me to import some target from the sdk to get the SDK version defined in our helix.proj?

Reply:
You could consider only importing the Microsoft.NET.Sdk's props file

```xml
<Import project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Reply:
There's a few ways to do this, but I think it's mostly done through global.json like here: https://github.com/dotnet/runtime/blob/master/global.json#L7-L9 which brings along and uses that SDK version, resulting in that property being set to that value, by that dotnet building thing (but for runtime they have special stuff to use the runtime they've built, already)

Reply:
Cool I actually just tried pushing that actually, including the sdk.props

Reply:
Looks like that works, but it does introduce a build warning (which is treated as an error), any ideas how to stop the double import?

```
 [error].dotnet\sdk\5.0.100-preview.6.20310.4\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.props(40,3): error MSB4011: "F:\workspace\_work\1\s\.dotnet\sdk\5.0.100-preview.6.20310.4\Current\Microsoft.Common.props" cannot be imported again. It was already imported at "F:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\5.0.0-beta.20316.1\tools\Microsoft.DotNet.Helix.Sdk.props (3,3)". This is most likely a build authoring error. This subsequent import will be ignored. [F:\workspace\_work\1\s\eng\helix\helix.proj]
```

Reply:
The Helix SDK (tools\Microsoft.DotNet.Helix.Sdk.props) imports Microsoft.Common.props and that's unconditional. Unfortunately, you'll need to suppress the warning using `<NoWarn>` or `<WarningsNotAsErrors>`.

Longer term, it might make sense to add a property in the Helix SDK to control the relevant `<Import>`.

Reply:
https://github.com/dotnet/arcade/pull/5747/files

Reply:
@HaoK please file an issue to remove whichever suppression you choose once @MattGal's longer-term fix is in and we get a new Helix SDK.